notaryproject/specifications
# Update COSE spec

Changes:
- Merge the latest changes from `main` branch
- Update COSE spec
- Add COSE support to the plugin spec

See also [comparing with `main`](https://github.com/notaryproject/notaryproject/compare/main..shizhMSFT:notaryproject:cose-envelope).
